apps/wallet/src/features/Widget/index.tsx

```diff
@@ -1,40 +1,36 @@
 import type { FC } from 'react';
-import type { ViewStyle } from 'react-native';
-import { View } from 'react-native';
-import { useSharedValue } from 'react-native-reanimated';
-import { StackHeader } from 'components/StackContainer';
-import { useSafeAreaInsets } from 'utils/hooks';
+import { useMemo } from 'react';
+import { withStackContainer } from 'components/StackContainer';
 
 import { extractWidgetComponent } from './internal';
 
 interface WidgetFeatureProps {
 	widgetId: string;
 	title: string;
-	style?: ViewStyle;
+	headerActive?: boolean;
 	onToggleDrawer: () => void;
 }
 
 export const WidgetFeature: FC<WidgetFeatureProps> = ({
 	widgetId,
 	title,
-	style,
+	headerActive = false,
```

Comment:
`isHeaderActive` or `noHeader`, and sounds like we got duplicated component/block?